### PR TITLE
assisted installer: replace postgres image

### PR DESCRIPTION
**Pull request by rccrdpccl**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.20
-  postgresql-12-centos7:
+  postgresql-12-c8s:
     name: postgresql-12-centos7
     namespace: edge-infrastructure
```
Comment (carbonin) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:52]: Given that this exists in our namespace does this really change anything?
Wouldn't we need to change this image elsewhere?

**Comment by adriengentil** [comment]
You also need to update https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_edge_infrastructure#L1 to mirror the new image.

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml (hunk 1 above)
Comment (rccrdpccl) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:52]: I guess we'd need changing the image mapping, but this should get things working and IMO we can change this in a second stage to have tests better reflecting reality.
Also it has been more than one year that we do not use edge-infrastructure image, so it was already doing it with a slightly different image

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-installer-master-edge-e2e-ai-operator-ztp

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 2]
```diff
@@ -373,7 +373,7 @@ tests:
           memory: 200Mi
     workflow: generic-claim
 - as: e2e-ai-operator-ztp
-  run_if_changed: ^(cmd/.*|data/.*|deploy/operator/.*|deploy/assisted-controller/.*|hack/.*|internal/.*|pkg/.*|tools/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum|swagger.yaml)$
+  run_if_changed: ^(cmd/.*|data/.*|deploy/operator/.*|deploy/assisted-controller/.*|hack/.*|internal/.*|pkg/.*|tools/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum|swagger.yaml|foobar.yaml)$
```
Comment (carbonin) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:376]: What is this?

Comment (rccrdpccl) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:376]: this is just a trick to force prow to allow me test some rehearsal jobs. before merging I'll revert this

Comment (carbonin) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:376]: Ah okay :+1: 

I just saw that "all tests passed" email so went to review :sweat_smile:

**Review by carbonin — APPROVED** [review]

**Pull request merged**